components/image.css

```diff
@@ -0,0 +1,17 @@
+.responsive-img-set{
+display: flex;
+width:40%;
+flex-wrap: wrap;
+gap: 2rem;
+margin: auto;
+}
+
+.img-resp {
+  width: 100%;
+  height: auto;
+  object-fit: cover;
+}
+
+.img-round {
+  border-radius: 100%;
```

Comment:
border-radius : 50% or border-radius :100% ??? 
Can you comfirm it?

Reply:
above 50% it becomes round